webui/ocw/views.py

```diff
@@ -45,6 +47,12 @@ def update_status(request):
     return redirect('instances')
 
 
+def cron(request):
+    update(request)
+    emailnotify.send_mail(request)
+    return HttpResponse('Done...')
```

Comment:
in case config is empty or something goes wrong we still sending done , right ?

Reply:
Hm don't get the point. The update() still takes place. And if a email was send or not, is also not visible. So what do you expect here.
For me done means, no exception occurred :)

We could redirect to `/instances` but this would just increase the traffic.